dev-util/pkgconf/pkgconf-1.0.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="git://github.com/pkgconf/pkgconf.git"
+	inherit autotools git-r3 multilib-minimal
+else
+	inherit multilib-minimal
+	SRC_URI="https://github.com/pkgconf/pkgconf/releases/download/${P}/${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~x64-freebsd ~x86-freebsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc64-solaris ~x64-solaris"
+fi
+
+DESCRIPTION="pkg-config compatible replacement with no dependencies other than ANSI C89"
+HOMEPAGE="https://github.com/pkgconf/pkgconf"
+
+LICENSE="BSD-1"
+SLOT="0"
+IUSE="+pkg-config strict"
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	pkg-config? (
+		!dev-util/pkgconfig
+		!dev-util/pkg-config-lite
+		!dev-util/pkgconfig-openbsd[pkg-config]
+	)"
+
+MULTILIB_CHOST_TOOLS=(
+	/usr/bin/pkgconf
+)
+
+src_prepare() {
+	[[ -e configure ]] || eautoreconf
```

Comment:
You are mixing two different conditions for relevant things. In other words:
1. autotools is inherited if ${PV}==9999,
2. but eautoreconf is called if configure does not exist.

Use the same condition throughout the whole file.